src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java

```diff
@@ -235,16 +235,26 @@ public ElasticsearchWriter build() {
   }
 
   public void write(Collection<SinkRecord> records) {
+    log.debug("Writing {} records to Elasticsearch", records.size());
```

Comment:
"duplicates the put() log message considering put() calls write()" - lev

Reply:
ack